elements/lisk-p2p/src/p2p.ts

```diff
@@ -406,6 +412,29 @@ export class P2P extends EventEmitter {
 						INVALID_CONNECTION_SELF_REASON,
 					);
 
+					// Ignore these steps for local network tests since all the IPs will be localhost
+					if (socket.remoteAddress !== '127.0.0.1') {
```

Comment:
Probably this is only helpful in the case of local?
maybe instead, when you check the nonce and found out it's urself, just add the the blacklist?